spec/ui_spec.rb

```diff
@@ -0,0 +1,38 @@
+require './lib/board.rb'
+include UI
+
+describe UI do
+     let(:board){ Board.new }
+  
+  describe "#decide_other_sign" do
+    it "takes the sign and returns the other one" do
+      expect(decide_other_sign('X')).to eql('O')
+    end
+  end
+
+
+  describe "#play_again" do
+    it "takes an answer as Y or N and returns it" do
+      stub(:gets) {"N"}
+      expect(play_again?).to eql('N')
+    end
+  end
+
+  describe "#choose_num" do
```

Comment:
Should have at least one test for when an out of bounds value is given. I strongly recommend that you do, as this is easy to test.

Reply:
The change for this method has been done